api/envoy/config/cluster/v3/cluster.proto

```diff
@@ -123,6 +123,8 @@ message Cluster {
   // only perform a lookup for addresses in the IPv6 family. If AUTO is
   // specified, the DNS resolver will first perform a lookup for addresses in
   // the IPv6 family and fallback to a lookup for addresses in the IPv4 family.
+  // If V4_PREFERRED is specified, the DNS resolver will first perform a lookup for addresses in the
+  // IPv4 family and fallback to a lookup for addresses in the IPv6 family.
```

Comment:
well, the funny thing is that in c-ares the two looksups _are_ sequential but in the apple resolver the lookups are concurrent. What I mean by fallback is: the callback target will only get v6 addresses _if_ there were NO v4 addresses to return.

Reply:
Heh. That makes sense. It might be nice to be explicit about "the callback target will only get v6 addresses if there were NO v4 addresses to return."

But I'm happy to defer to you.